### DISCUSSION TITLE
Min-height use cases

### DISCUSSION THREAD

**Discussion by AnnaYuS** [tailwindlabs/tailwindcss#3330] [Help]
I've noticed that by default Tailwind offers only 3 values for min-height. I was wondering if it's a bad practice to use other values for min-height?

In the project that I'm working on there is a Vue component for a modal that sometimes contains 1 line of text and sometimes  dynamic content of unknown height. I was wondering if there is a way to not use `min-height` (if it's a bad practice) to control the height of the modal content? A regular height will make a scroll to appear if the content is taller than the height value. I need the dialog height to grow with the content till it reaches max-height.

Example: https://play.tailwindcss.com/S9pXhe3fk6

**Comment by excessivecoding** [comment]
I don't think this is a bad practice to add `min-height` value if you need them.

`min-height` value can be highly dependent on the project compare to `height` value which would explain why Tailwind.css doesn't ship with too much value out of the box (This is just a guess, I'm not sure of that statement).

Tailwind.css is made easily extendable so don't hesitate to add what you need
